.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```

Comment:
Did you mean to check this in? This doesn't match the .ruby-version file.

Reply:
ah, hm yeah it got in accidentally

Reply:
removed